app-crypt/scrypt/scrypt-1.3.1.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A simple password-based encryption utility using scrypt key derivation function"
+HOMEPAGE="http://www.tarsnap.com/scrypt.html"
+SRC_URI="http://www.tarsnap.com/scrypt/${P}.tgz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DOCS=( FORMAT )
+
+IUSE="test"
```

Comment:
This doesn't add anything because we don't have any test dependencies. It's only needed when we're passing something through to the build system or checking deps.

Reply:
Ah, I see, thanks for explaining